### DISCUSSION TITLE
BPX61 (or other) window

### DISCUSSION THREAD

**Discussion by L7dog** [ozel/DIY_particle_detector#44] [Ideas]
The tips suggested using capacitor foil as a thin window.
If you know an artist (or go to a hobby store/Micheals, etc.), faux silver leaf is ultra thin and lightproof - made from aluminum, of course.
Have a milligram scale and a 1x1cm piece won't even register.

**Comment by ozel** [comment]
very nice suggestion, thanks! Some of those "Imitation Silver Leaf" products even specify the thickness, which is awesome for estimating energy loss.
